ci/scripts/utils/pr_list_database.py

```diff
@@ -3,28 +3,27 @@
 import sys
 import os
 from wxflow import SQLiteDB, SQLiteDBError
+from githubpr import GitHubPR
 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, REMAINDER
 
 
 def full_path(string):
     """
     full_path Get the absolute path of a file or directory.
-
```

Comment:
Why did you remove these spacer lines? Seems like it would make the docblock more difficult to read.